### PR TITLE
[15.0][MIG] base_external_dbsource_mysql: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @carlosdauden and @sergio-teruel can you review it?

@Tecnativa TT37274

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]

File: base_external_dbsource_mysql/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+* Daniel Reis <dreis.pt@hotmail.com>
+* Maxime Chambreuil <maxime.chambreuil@savoirfairelinux.com>
+* Gervais Naoussi <gervaisnaoussi@gmail.com>
+* Dave Lasley <dave@laslabs.com>
+* David Alonso <david.alonso@solvos.es>
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+    * Sergio Teruel
```
Comment (carlosdauden) [inline base_external_dbsource_mysql/readme/CONTRIBUTORS.rst]:
```suggestion
* `Tecnativa <https://www.tecnativa.com>`_:

  * Sergio Teruel
```

**Review by carlosdauden — APPROVED** [review]
Minor comment

File: base_external_dbsource_mysql/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (carlosdauden) [inline base_external_dbsource_mysql/readme/CONTRIBUTORS.rst]: Keep original order:

```suggestion
* `Tecnativa <https://www.tecnativa.com>`_:

  * Sergio Teruel
* David Alonso <david.alonso@solvos.es>
```

**Comment by victoralmau** [comment]
It's ready to merge, right?

**Pull request merged**